### PR TITLE
allow assets to be deleted

### PR DESCRIPTION
**Pull request by akash1810**
If the asset isn't active, allow it to be deleted.

asset-delete

### CODE REVIEW

**Comment by mbarton** [comment]
I think this would be confusing since it does not also delete the YouTube video (or set it as private)

**Comment by mbarton** [comment]
Also let's make it so it has to be clicked twice like composer deletions. It's precariously close to the Activate button and it would be difficult for a user to re-upload since YouTube would see it as a dupe

**Comment by akash1810** [comment]
Updated with generic `DeleteButton` component.

update

File: public/video-ui/styles/components/_buttons.scss [hunk 1]
```diff
@@ -46,13 +29,33 @@ button {
...
   }
-  
+
   &--list {
     width: 80px;
     padding: 7px;
   }
 }
 
+.button__secondary {
```
Comment (akash1810) [inline public/video-ui/styles/components/_buttons.scss]: ahh cascading style... `.button__secondary ` needs to come after `.btn` to win

**Pull request merged**